packages/design-system/src/components/Table/Table.jsx

```diff
@@ -75,12 +75,19 @@ export class Table extends React.PureComponent {
 
   renderChildren() {
     return React.Children.map(this.props.children, (child) => {
-      // Extend props on TableCaption before rendering.
-      if (this.props.scrollable && isTableCaption(child)) {
+      if (isTableCaption(child)) {
+        // Extend props on TableCaption before rendering.
+        if (this.props.scrollable) {
+          return React.cloneElement(child, {
+            _id: this.captionID,
+            _scrollActive: this.state.isTableScrollable,
```

Comment:
Could we rename `this.state.isTableScrollable` to be `scrollActive` to avoid confusion?